src/BenchmarkDotNet/Toolchains/InProcess/NoEmit/BenchmarkActionFactory_Implementations.cs

```diff
@@ -14,6 +15,9 @@ namespace BenchmarkDotNet.Toolchains.InProcess.NoEmit
     /// <summary>Helper class that creates <see cref="BenchmarkAction"/> instances. </summary>
     public static partial class BenchmarkActionFactory
     {
+        // Not available in netstandard2.0, so just use the value casted to enum.
+        public const MethodImplOptions AggressiveOptimization = (MethodImplOptions) 512;
```

Comment:
Could we move this constant to a better namespace? `BenchmarkDotNet.Toolchains.InProcess.NoEmitBenchmarkActionFactory` looks like a strange place for such an important constant.

Reply:
Do you have a suggestion? I wasn't sure of a best place to put it.

Reply:
What about `BenchmarkDotNet.Portability`?